src/plugins/data/common/search/tabify/tabify.ts

```diff
@@ -139,7 +139,7 @@ export function tabifyAggResponse(
   const write = new TabbedAggResponseWriter(aggConfigs, respOpts || {});
   const topLevelBucket: AggResponseBucket = {
     ...esResponse.aggregations,
-    doc_count: esResponse.hits.total,
+    doc_count: esResponse.hits?.total,
```

Comment:
not sure if this is intended as part of this pr

Reply:
It is, hits are optional in all other places where they are used.
For example `src/plugins/data/common/search/expressions/utils/courier_inspector_stats.ts`

And not having the `?` here  was breaking some new tests I wrote.